lib/LaTeXML/Core/Document.pm

```diff
@@ -1595,6 +1595,7 @@ sub setNodeBox {
 
 sub getNodeBox {
   my ($self, $node) = @_;
+  return unless $node;
```

Comment:
i ended up blowing up on this while prototyping, so kept the guard in just in case.